pkg/kubelet/cm/container_manager_linux.go

```diff
@@ -193,6 +194,43 @@ func validateSystemRequirements(mountUtil mount.Interface) (features, error) {
 	return f, nil
 }
 
+func validateSwapConfiguration(fs fs.FS, failSwapOn bool) error {
```

Comment:
this is a good change! I think in the future we should do a trivial code extraction + test addition (chicken and egg problem, we need the extraction to be able to add the tests) and only then improve the extracted code

Reply:
👍